MAUI/TabView/Selection-Indicator-Customization.md

```diff
@@ -163,4 +163,22 @@ The [IndicatorWidthMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Tab
 
 {% endtabs %}
 
-![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
\ No newline at end of file
+![Selection Indicator Background.](images/IndicatorWidthMode_Stretch.png) 
+
+## Indicator's corner radius
+
+The selection Indicator can be customized using `CornerRadius` property, which is type of CornerRadius.
```

Comment:
You can customize the corner radius of the selection indicator using the CornerRadius property in the Tab View.

Reply:
Added changes